HelpSource/Classes/A2K.schelp

```diff
@@ -1,17 +1,41 @@
-class:: A2K
-summary:: Audio to control rate converter.
+title:: A2K
+categories:: UGens>Conversion
+summary:: Audio to control rate converter
 related:: Classes/K2A
-categories::  UGens>Conversion
 
 
-Description::
+
+
+
+
+
+
+
+
+description::
+
+
+
```

Comment:
are you sure this script is working as intended? The PR seems to add lots of blank lines.

Reply:
The number of blank lines is as intended.
Please let me know the ideal numbers if you do not like this. It is very easy to change.